crates/next-core/js/src/overlay/internal/container/Errors.tsx

```diff
@@ -157,18 +163,34 @@ export function Errors({ errors }: ErrorsProps) {
     []
   );
 
+  const hasIssues = issues.length > 0;
+  const hasIssueWithError = issues.some((issue) =>
+    ["bug", "fatal", "error"].includes(issue.severity)
+  );
+
   const hasErrors = errors.length > 0;
   const hasServerError = readyErrors.some((err) =>
     ["server", "edge-server"].includes(getErrorSource(err.error) || "")
   );
-  const isClosable = !isLoading && !hasServerError;
+  const isClosable = !isLoading && !hasIssueWithError && !hasServerError;
+
+  const defaultTab =
+    hasIssueWithError || !hasErrors
+      ? TabId.TurbopackIssues
+      : TabId.RuntimeErrors;
 
-  const defaultTab = TabId.RuntimeErrors;
   const [selectedTab, setSelectedTab] = React.useState<string>(defaultTab);
 
+  React.useEffect(() => {
+    if (defaultTab === TabId.TurbopackIssues) {
+      setSelectedTab(TabId.TurbopackIssues);
+      setDisplayState("fullscreen");
+    }
+  }, [defaultTab]);
```

Comment:
Could this force-set `selectedTab`/`displayState` instead of updating state? (like the suggestion above)

Reply:
This one is a bit stupid to be fair, it's supposed to check if the `defaultTab` from the above condition **changed** to `TabId.TurbopackIssues` (there are issues now and there weren't before) and then open the overlay or change the tab

But can probably move the `displayState` out, as that should be forced